modules/cluster-logging-dashboards-es.adoc

```diff
@@ -0,0 +1,196 @@
+|The time taken to complete search requests on the selected Elasticsearch node. Search latency can be affected by many factors. A growing trend indicates a resource capacity shortage in the instance.
+
+|Documents count (with replicas) 
+|The number of Elasticsearch documents stored on the selected Elasticsearch node, including documents stored in both the primary shards and replica shards that are allocated on the node.
+
+|Documents deleting rate 
+|The number of Elasticsearch documents being deleted from any of the index shards allocated to the selected Elasticsearch node.
+
+|Documents merging rate 
+|The number of Elasticsearch documents being merged in any of index shards allocated to the selected Elasticsearch node.
+
+|===
+
+Elasticsearch node fielddata::
+
+_Fielddata_ is a query-time in-memory data structure for text fields that is built by Elasticsearch at the index-segment level and kept in the JVM Heap. Because fielddata building is an expensive operation, Elasticsearch caches the fielddata structures. Elasticsearch can evict a fielddata cache when the underlying index segment is deleted or merged, or if there is not enough JVM HEAP memory for all the fielddata caches. 
+
+The *Logging/Elasticsearch Nodes* dashboard contains the following charts about Elasticsearch fielddata.
+
+.Elasticsearch node fielddata charts
+[options="header"]
+|===
+|Metric|Description
+
+|Fielddata memory size 
+|The amount of JVM Heap used for the fielddata cache on the selected Elasticsearch node.
+
+|Fielddata evictions 
+|The number of fielddata structures that were deleted from the selected Elasticsearch node. 
+
+|===
+
+Elasticsearch node query cache::
+
+If the data stored in the index does not change, search query results are cached in a node-level query cache for reuse by Elasticsearch. 
+
+The *Logging/Elasticsearch Nodes* dashboard contains the following charts about the Elasticsearch node query cache.
+
+.Elasticsearch node query charts
+[options="header"]
+|===
+|Metric|Description
+
+|Query cache size 
+|The total amount of memory used for the query cache for all the shards allocated to the selected Elasticsearch node.
+
+|Query cache evictions 
+|The number of query cache evictions on the selected Elasticsearch node.
+
+|Query cache hits 
+|The number of query cache hits on the selected Elasticsearch node.
+
+|Query cache misses 
+|The number of query cache misses on the selected Elasticsearch node.
+
+|===
+
+Elasticsearch index throttling::
+
+When indexing documents, Elasticsearch stores the documents in index segments, which are physical representations of the data. At the same time, Elasticsearch periodically merges smaller segments into a larger segment as a way to optimize resource use. If the indexing is faster then the ability to merge segments, the merge process gets behind. To prevent this situation, Elasticsearch throttles indexing, typically by reducing the number of threads allocated to indexing down to a single thread. 
```

Comment:
```suggestion
When indexing documents, Elasticsearch stores the documents in index segments, which are physical representations of the data. At the same time, Elasticsearch periodically merges smaller segments into a larger segment as a way to optimize resource use. If the indexing is faster then the ability to merge segments, the merge process does not complete quickly enough <and why this is a problem>. To prevent this situation, Elasticsearch throttles indexing, typically by reducing the number of threads allocated to indexing down to a single thread. 
```